lib/puppet/file_system/file_impl.rb

```diff
@@ -14,7 +14,7 @@ def assert_path(path)
     if path.respond_to?(:to_str)
       Pathname.new(path)
     else
-      raise ArgumentError, "FileSystem implementation expected Pathname, got: '#{path.class}'"
+      raise ArgumentError, _("FileSystem implementation expected Pathname, got: '#{path.class}'")
```

Comment:
Ah, another subdirectory accidentally lumped in the wrong PR. Not going to split it out, I'll just fix it up here.